flixel/input/FlxPointer.hx

```diff
@@ -15,6 +15,7 @@ class FlxPointer
 	
 	private var _globalScreenX:Int = 0;
 	private var _globalScreenY:Int = 0;
+	private static var _cachedPoint:FlxPoint = new FlxPoint();
```

Comment:
I'm trying to make it obvious that these are static cached values (to prevent confusion or accidental deletion), but I am open to changing this to better fit existing conventions if you'd prefer.